queue/queue.py

```diff
@@ -1,20 +1,22 @@
+
 import sys
-#sys.path.append('../linked_list')
-#from linked_list import LinkedList
+
```

Comment:
Not a requirement, but how could you implement storage with a linked list instead of an array?

Reply:
bring in a linked list or double linked list. Use the methods I created to add and remove items to the list